accounts.go

```diff
@@ -79,6 +92,26 @@ func (c *Client) AccountUpdate(ctx context.Context, profile *Profile) (*Account,
 	if profile.Note != nil {
 		params.Set("note", *profile.Note)
 	}
+	if profile.Locked != nil {
+		params.Set("locked", strconv.FormatBool(*profile.Locked))
+	}
+	if profile.Fields != nil {
+		for idx, field := range *profile.Fields {
```

Comment:
Mastodon explicitly states only 4 entries are allowed, but I figured it’s best letting the server complain.